examples/package.json

```diff
@@ -4,17 +4,14 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "cross-env HTTPS=true react-app-rewired start",
-    "start:e2e-server": "cross-env PORT=3000 HTTPS=false BROWSER=none react-app-rewired start",
-    "build": "react-app-rewired build",
```

Comment:
Can we make a note that examples do not need to be build with the changes from here!